src/services/GraphQLService.ts

```diff
@@ -108,7 +111,12 @@ class GraphQLService {
         if (!query || query.length === 0) {
             throw new Error('GraphQLService query expects a GraphQL query as its first argument');
         }
-        return ApiBase.queryGraph(query.replace(/\s\s+/g, ' ').trim(), variables);
+
+        if (variables || query.trim().startsWith('{')) {
```

Comment:
Fragments cause query to error outright. Needs way to detect fragments and prepend them/or not cache them.